src/components/modal/modal.js

```diff
@@ -534,7 +546,10 @@ export default {
       }
       this.emitEvent(hideEvt)
       // Hide if not canceled
-      if (hideEvt.defaultPrevented || !this.is_visible) {
+      if (hideEvt.defaultPrevented) {
+        return
+      }
+      if (!this.is_visible) {
         return
```

Comment:
Why break this test into two separate if statements?

Reply:
Sorry. I changed it so that it is convenient for debugging and I forgot to revert it. I will revert it.